lib/variable.js

```diff
@@ -79,7 +79,7 @@ module.exports = function environmentVariable (name, value) {
         );
       }
 
-      return Boolean(val);
+      return Boolean('false' === val ? 0 : val);
```

Comment:
@itavy unfortunately this isn't so much a JS issue as an issue of how people on systems declare booleans. Some people use 0 and 1 instead of true and false - although, like you, I prefer to be explicit so the module can simply not support that unless someone needs in the future.

In terms of upper/lower case: some people will use "FALSE" and others will use "false", so it's best to support this. If you can do this then I am happy to merge 👍